src/shared/components/Banner/Banner.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+
+import { Banner } from './Banner';
+import { BANNER_LINK } from './Banner.const';
+
+describe('Banner component tests', () => {
```

Comment:
No need to add `component tests`, just source file name is enought.

Reply:
fixed